reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -571,6 +571,7 @@ <h3 class="">Packages</h3>
             <td><a href="#Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0-pkg-1">1</a></td><td>Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0</td><td></td><td><em>Detected Licenses:</em>
               <dl>
                 <dd>
+                  <div>GPL-2.0-only WITH Classpath-exception-2.0 AND BSD-3-Clause OR MIT (<a href="https://github.com/oss-review-toolkit/ort/tree/master/analyzer/src/funTest/assets/projects/synthetic/gradle/lib/src/code.cpp#L1">link</a> to the location)</div>
```

Comment:
> This shows that it would be important for the reporter to know that the license was curated so that it does not create a broken link, and instead shows the curation message.

Showing the curation message was not implemented before in the WebApp and isn't implemented in the static HTML report either. Would you be ok with keeping it out of scope here as well?

> Otherwise you should make sure that the link points to a file location that actually contains the license expression.

That's impossible, isn't it? Maybe I'm not getting it. Is the comment still valid?

Reply:
It's not impossible, the other links in the test report also point to valid locations. But it would also not be nice to modify the linked files for this test, because they actually belong to `GradleTest`. But I think we should at least document the issue that links can be either broken or point to a license different from the one shown in the report if the finding was curated, maybe with a TODO comment in the `StaticHtmlReporter` next to the code generating the link, or with a GitHub issue.

Reply:
As discussed offline I've filed an issue for improving the finding curation handling in the reports, see #2605.